src/services/users.service.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+let users = [];
```

Comment:
Why do we need let in this case?
Don`t see any reason why we don't use const for this array.
Try rewriting using const. We avoid not needed assignments.

Reply:
I used let instead of const for deleting users and expenses from array, ok, I'll change to const